packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -65,14 +65,18 @@ class PointerEventConverter {
     );
   }
 
-  /// Expand the given packet of pointer data into a sequence of framework
+  static String _formatErrorMessage(ui.PointerChange change, String msg) => 'Invalid event \'$change\': $msg';
```

Comment:
msg -> message
(we don't use abbreviations)

Reply:
fixed